ui/src/components/ChatComponent.vue

```diff
@@ -48,16 +73,33 @@
 </template>
 
 <script setup lang="ts">
-import { ref, computed, Ref } from 'vue';
+import { ref, computed, Ref, onMounted } from 'vue';
 import { date } from 'quasar';
-import { sendQuestion } from 'src/api/index';
+import { sendQuestion, getAppSettings } from 'src/api/index';
 import {
   DeepshoreChatMessage,
   DeepshoreChatRequest,
   DeepshoreChatResponse,
 } from 'src/types/chat';
+import { AppSettings } from 'src/types/appsettings';
+
 
 // data
+const chatScrollRef = ref('chatScrollRef')
+let toolbar = ref(true)
+
+const defaultAppSettings: AppSettings = {
+  title: 'Knowledge Chatbot',
+  disclaimer: 'Lorem ipsum<br />dolorem' ,
+  icon: 'deepshore.png'
+}
+
+const appSettings: Ref<AppSettings> = ref(defaultAppSettings)
+
+onMounted(async () => {
+  appSettings.value = await getAppSettings()
+})
+
 let message = ref('');
 let chat: Ref<Array<DeepshoreChatMessage>> = ref([]);
 let blockedAPI = ref(false);
```

Comment:
errMessages könnte man auf einen Festen String kürzen. Ich hatte da nur mehrere Messages, weil die API öfter einen Internal Server Error zurückgegeben hat. mMn lieber etwas in Richtung:
'Leider ist da ein Fehler aufgetreten. Bitte versuche es doch noch einmal. Vielleicht mit einer anderen Formulierung?'

Dann den Rest weghauen. `getRandomInt` ebenfalls weg

Reply:
Habe das gemacht und auch das errorhandling optimiert wenn ein AxiosError zurück kommt....